bundle/config/mutator/override_compute.go

```diff
@@ -28,10 +28,10 @@ func overrideJobCompute(j *resources.Job, compute string) {
 			task = &task.ForEachTask.Task
 		}
 
-		if task.NewCluster != nil || task.ExistingClusterId != "" || task.ComputeKey != "" || task.JobClusterKey != "" {
+		if task.NewCluster != nil || task.ExistingClusterId != "" || task.EnvironmentKey != "" || task.JobClusterKey != "" {
 			task.NewCluster = nil
 			task.JobClusterKey = ""
-			task.ComputeKey = ""
+			task.EnvironmentKey = ""
```

Comment:
This rename / deprecation happened on SDK side which is potentially a breaking change on CLI / DABs side as well

Reply:
ComputeKey was in private preview and deprecated in favour of EnvironmentKey so safe to make the change